lib/shopify_api/resources/base.rb

```diff
@@ -4,6 +4,7 @@ module ShopifyAPI
   class Base < ActiveResource::Base
     class InvalidSessionError < StandardError; end
     extend Countable
+    self.format = :json
```

Comment:
Why is this necessary? I thought in the tests we set this to json, but in the wild it could totally be using XML here no?